i18n/translations/en/form-builder.json

```diff
@@ -509,6 +509,20 @@
       "infoBox": "Which date format should you use?",
       "infoLine1": "Canada’s date standard format is YYYY-MM-DD and should be your first choice unless there is a reason to choose a different format.",
       "infoLine2": "For example, when asking for a date exactly as it’s shown on a passport, card or other document, make the fields match the format of the original. This makes it easier for users to copy the date across accurately."
+    },
+    "apiOptionsSection": {
+      "title": "Customize API data attributes",
+      "description": "Modify how your form fields get encoded for the API integration."
+    },
+    "questionId": {
+      "title": "Question ID",
+      "description": "Unique value to consistently refer to a form element so that they can match across republished form versions, data structures and systems.",
```

Comment:
```suggestion
      "description": "Set a custom unique value to consistently refer to a form element so that they can match across republished form versions, data structures, and systems.",
```